README.md

```diff
@@ -2,3 +2,39 @@ Instant Share
 =============
 
 Instantly share images, videos, and other files on the web.
+
+Features
+--------
+
+-	**Instant links**
+	-	When you share via Instant Share, you get a shareable link in your clipboard right away, so you can paste it in a conversation, in a forum post, or anywhere. That link is usable right away because Instant Share streams both the uploads and downloads in the background.
```

Comment:
... downloads in the background, allowing downloads to begin before you finish uploading. <-- maybe?